pkgs/development/python-modules/snowflake-connector-python/default.nix

```diff
@@ -48,6 +49,7 @@ buildPythonPackage rec {
   '';
 
   nativeBuildInputs = [
+    packaging
```

Comment:
Is this required? (I wasn't able to find a direct dependency on this in a build script.)

Reply:
https://github.com/snowflakedb/snowflake-connector-python/blob/1380e41cd2088f0c83af0cf45010d68aac96b366/setup.cfg#L55C9-L55C9

Reply:
I think that means it's required in `propagatedBuildInputs`, and you added it there, but it shouldn't be needed (at least not directly) in `nativeBuildInputs`.

Reply:
Cool, let's try without.

Also realised I could use the `pythonRelaxDepsHook` for `platformdirs`